ssp_author_demo/README.md

```diff
@@ -1,49 +1,67 @@
 
-The trestle project has been setup with a catalog and profile from NIST using 800-53. This was used to generate the markdown directory.
+This demo requires trestle version 3.x.x
 
-SSP generate is used to generate the markdown files from a profile and its imported catalogs and profiles.  Prompts are provided in the markdown for
-each control where an implementation response is required, corresponding to parts in the control statement.
+## Steps
 
-This demo requires trestle version 1.0.x
+### Download this repo
 
-## Steps to recreate setup
+```
+> cd
+> mkdir git
+> cd git
+> git clone https://github.com/oscal-compass/compliance-trestle-demos.git
+```
 
-### Initial import
+### Install compliance trestle, ideally in a python virtual environment, and create a trestle workspace
 
-- trestle init was run
-- The [OSCAL-content](https://github.com/usnistgov/OSCAL-content) repository was cloned.
-- The [NIST 800-53 catalog](https://github.com/usnistgov/oscal-content/blob/master/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_catalog.json) was imported with `trestle import -f {path to catalog} -o 800-53`
-  - The catalog will be inserted within `./catalogs/800-53/`
-- The [NIST 800-53 LOW profile](https://github.com/usnistgov/oscal-content/blob/master/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_LOW-baseline_profile.json) was imported with trestle import -f {path to profile} -o 800-53-low\`
-  - The profiles will be inserted within `./profiles/800-53-low/`
-- The profile is updated so the import href points within the trestle project to `trestle://catalogs/800-53/catalog.json`
+```
+> cd
+> python -m venv venv.compliance-trestle-demos
+> source venv.compliance-trestle-demos/bin/activate
+> cd git/compliance-trestle-demos
+> make install
+> cd ssp_author_demo
+> trestle init
+```
+
+### Download example catalog([NIST 800-53 catalog](https://github.com/usnistgov/oscal-content/blob/master/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_catalog.json)) and profile([NIST 800-53 LOW profile](https://github.com/usnistgov/oscal-content/blob/master/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_LOW-baseline_profile.json))
+
+```
+> cd
+> git clone https://github.com/usnistgov/OSCAL-content
+> cd git/compliance-trestle-demos/ssp_author_demo
+> trestle import -f ~/OSCAL-content/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_catalog.json -o 800-53
+> trestle import -f ~/OSCAL-content/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_LOW-baseline_profile.json -o 800-53-low
+```
+
+The example catalog and profile are inserted within `./catalogs/800-53/` and `./profiles/800-53-low/`.\
+Note, the `url` in the example profile doesn't correctly link to the imported catalog.
+Select the imported profile, and scroll down to the `resources` section and replace the json media-type's `href` value to: `trestle://catalogs/800-53/catalog.json`
```

Comment:
I was thinking of using trestle href command to change the href, but the command failed to execute on the example profile. It output an error message: 'NoneType' object has no attribute 'exists': 'NoneType' object has no attribute 'exists', and I couldn't figure out where went wrong.